lib/internal/cluster/master.js

```diff
@@ -70,6 +79,15 @@ cluster.setupMaster = function(options) {
   assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
          `Bad cluster.schedulingPolicy: ${schedulingPolicy}`);
 
+  const hasDebugArg = process.execArgv.some((argv) => {
```

Comment:
There's a new undocumented property `process._inspectorEnbale` that tells you that 😃